modules/controlfreec/1.0/config/default.yaml

```diff
@@ -0,0 +1,72 @@
+lcr-modules:
+    
+    controlfreec:
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            # sample_bam: "__UPDATE__"
+            # sample_bai: "__UPDATE__"
+            sample_bam: "/projects/rmorin/projects/tumour_contam/lcr-modules/demo/data/{sample_id}.bam"
```

Comment:
Change these back to the UPDATE lines (i.e. comment these and uncomment the ones above. You also don't need the full path and should just start with "data/". Then the example will be more generic and this will be more portable.

Reply:
Changed